test/spec/provider/camunda/element-templates/cmd/ChangeElementTemplateHandlerSpec.js

```diff
@@ -2664,93 +2887,539 @@ describe('element-templates - ChangeElementTemplateHandler', function() {
+
+        expect(error).to.exist;
+        expect(error.get('id').indexOf('error-1')).to.equal(0); // start with binding error ref
+        expect(error.get('errorCode')).to.equal('error-code');
+        expect(error.get('name')).to.equal('error-name');
+        expect(error.get('errorMessage')).to.eql('error-message');
+      }));
+
+
+      it('properties unchanged', inject(function(elementRegistry) {
+
+        // given
+        var serviceTask = elementRegistry.get('ServiceTask_1');
+
+        var oldTemplate = {
+          properties: [
+            {
+              value: 'error-expression-old-value',
+              binding: {
+                type: 'camunda:errorEventDefinition',
+                errorRef: 'error-1'
+              }
+            },
+          ],
+          scopes: [
+            {
+              id: 'error-1',
+              type: 'bpmn:Error',
+              properties: [
+                {
+                  value: 'error-code',
+                  binding: {
+                    type: 'property',
+                    name: 'errorCode'
+                  }
+                },
+                {
+                  value: 'error-message',
+                  binding: {
+                    type: 'property',
+                    name: 'camunda:errorMessage'
+                  }
+                },
+                {
+                  value: 'error-name',
+                  binding: {
+                    type: 'property',
+                    name: 'name'
+                  }
+                }
+              ]
+            }
+          ]
+        };
+
+        var newTemplate = {
+          properties: [
+            {
+              value: 'error-expression-new-value',
+              binding: {
+                type: 'camunda:errorEventDefinition',
+                errorRef: 'error-1'
+              }
+            },
+          ],
+          scopes: [
+            {
+              id: 'error-1',
+              type: 'bpmn:Error',
+              properties: [
+                {
+                  value: 'error-code',
+                  binding: {
+                    type: 'property',
+                    name: 'errorCode'
+                  }
+                },
+                {
+                  value: 'error-message',
+                  binding: {
+                    type: 'property',
+                    name: 'camunda:errorMessage'
+                  }
+                },
+                {
+                  value: 'error-name',
+                  binding: {
+                    type: 'property',
+                    name: 'name'
+                  }
+                }
+              ]
+            }
+          ]
+        };
+
+        changeTemplate(serviceTask, oldTemplate);
+
+        // when
+        changeTemplate(serviceTask, newTemplate, oldTemplate);
+
+        // then
+        var errorEventDefinition = findErrorEventDefinition(serviceTask, 'error-1');
+
+        expect(errorEventDefinition).to.exist;
```

Comment:
Should we not have an expect here?:

`expect(errorEventDefinition.get('expression')).to.eql('error-expression-new-value');`

Reply:
We should, probably a missed WIP state while adding those bunches of tests 😄